roles/kubernetes-apps/policy_controller/calico/templates/calico-kube-cr.yml.j2

```diff
@@ -107,4 +107,5 @@ rules:
       - update
       # watch for changes
       - watch
+      - list
```

Comment:
Add this under the `- get`
https://github.com/projectcalico/calico/blob/v3.29.2/charts/calico/templates/calico-kube-controllers-rbac.yaml